python/ray/air/tests/test_keras_callback.py

```diff
@@ -13,6 +17,114 @@
 )
 
 
+class TestReportCheckpointCallback:
```

Comment:
Why do we need the class here?

Reply:
We don't. It was a stylistic choice to group the `ReportCheckpointCallback` tests together. Without it, we may need to include something like `report_checkpoint_callback` in the test names to distinguish them from the old tests.

Reply:
Ok that's fine with me